### DISCUSSION TITLE
Current development version stability

### DISCUSSION THREAD

**Discussion by isolated-matrix** [MDAnalysis/mdanalysis#4791] [General]
Hi there,

I have been waiting for v2.8.0 to release for a while (ever since I saw that it was going to include support for elements from the periodic table), but it is taking a lot longer than planned (which is fair enough—I'm not accusing the developers of negligence!) and I would like to use those features sooner rather than later.

I was just wondering if the current development version of MDA is stable enough to run without crashing? I ask because I assume there were some unforeseen problems that have delayed the official release, but I do not know whether these are fatal or are just tweaks that need to be made before release.

**Comment by IAlibay** [comment]
@isolated-matrix we are currently in the process of releasing 2.8.0, probably at some point this week. The delay is purely down to lack of time.

**Reply by isolated-matrix** [reply]
I perfectly understand—things always come up, and programming can be particularly painful and time-consuming (at least, in my limited experience). I don't want to come across as being pushy, impatient, or self entitled in any way, I just wanted to check whether installing the development version would be likely to give me results faster or not.

**Reply by orbeckst** [reply]
By now, 2.8.0 has been out for a week ;-), see https://www.mdanalysis.org/2024/11/22/release-2.8.0/

To answer your original question: *develop* is _supposed_ to be stable and usable but we will never officially recommend you using it because we _may_ have to change things at short notice during development (or even undo changes) and for **reproducible science** you want to be able to give specific versions of software used.
